build_tools/customize_build.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
Can you document what this script is doing at a high level?

Reply:
Will add more comments, yep.